### DISCUSSION TITLE
Need Urgent Help. Livewire@2 and AlpineJS@3.9, alpine data not updating in livewire template change.

### DISCUSSION THREAD

**Discussion by sahakyanArt** [livewire/livewire#4762] [1. Help]
Question is that alpine data not updating after livewire is rendered with a different template. Init function is work but inputs values dose not change. Why? How to solve?
Example see here https://laravelplayground.com/#/snippets/fd7f4773-5312-466f-a217-9d842f617e99

**Comment by joshhanley** [comment]
@sahakyanArt can you please provide some instructions as to what we are meant to be doing on your playground and what isn't working/ what should it do?

Thanks!

**Reply by sawirricardo** [reply]
@joshhanley probably related to this https://github.com/livewire/livewire/issues/920#issuecomment-623680787
